misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/TesterinaConstants.java

```diff
@@ -101,4 +101,18 @@ public class TesterinaConstants {
     public static final int IDENTIFIER_START_INDEX = 1;
     public static final int IDENTIFIER_END_INDEX = 5;
 
+    //class to identify the indices of the BTestMain args
+    public static class RunTimeArgs {
+        public static final int IS_FAT_JAR_EXECUTION = 0;
```

Comment:
Where are these being used?

Reply:
This class is used in C2C side. Specifically addDockerTestCMDArgs function in line no 344 of [DockerGenerator.java](https://github.com/ballerina-platform/module-ballerina-c2c/blob/6a9e1b23efcb508a7624bf0914e7e87f402205f5/compiler-plugin/src/main/java/io/ballerina/c2c/utils/DockerGenerator.java)